Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -15,509 +17,693 @@
+    "filePath": "Path",
+    "fileSize": "Size",
+    "md5Checksum": "MD5",
+    "sha256Checksum": "SHA256",
+    "osHostName": "Hostname",
 }
 
 CODE42_FILE_TYPE_MAPPER = {
-    'SourceCode': 'SOURCE_CODE',
-    'Audio': 'AUDIO',
-    'Executable': 'EXECUTABLE',
-    'Document': 'DOCUMENT',
-    'Image': 'IMAGE',
-    'PDF': 'PDF',
-    'Presentation': 'PRESENTATION',
-    'Script': 'SCRIPT',
-    'Spreadsheet': 'SPREADSHEET',
-    'Video': 'VIDEO',
-    'VirtualDiskImage': 'VIRTUAL_DISK_IMAGE',
-    'Archive': 'ARCHIVE'
+    "SourceCode": "SOURCE_CODE",
+    "Audio": "AUDIO",
+    "Executable": "EXECUTABLE",
+    "Document": "DOCUMENT",
+    "Image": "IMAGE",
+    "PDF": "PDF",
+    "Presentation": "PRESENTATION",
+    "Script": "SCRIPT",
+    "Spreadsheet": "SPREADSHEET",
+    "Video": "VIDEO",
+    "VirtualDiskImage": "VIRTUAL_DISK_IMAGE",
+    "Archive": "ARCHIVE",
 }
 
-SECURITY_EVENT_HEADERS = ['EventType', 'FileName', 'FileSize', 'FileHostname', 'FileOwner', 'FileCategory', 'DeviceUsername']
-SECURITY_ALERT_HEADERS = ['Type', 'Occurred', 'Username', 'Name', 'Description', 'State', 'ID']
+SECURITY_EVENT_HEADERS = [
+    "EventType",
+    "FileName",
+    "FileSize",
+    "FileHostname",
+    "FileOwner",
+    "FileCategory",
+    "DeviceUsername",
+]
+SECURITY_ALERT_HEADERS = ["Type", "Occurred", "Username", "Name", "Description", "State", "ID"]
+
+
+def _get_severity_filter_value(severity_arg):
+    """Converts single str to upper case. If given list of strs, converts all to upper case."""
+    if severity_arg:
+        return (
+            [severity_arg.upper()]
+            if isinstance(severity_arg, str)
+            else list(map(lambda x: x.upper(), severity_arg))
+        )
+
+
+def _create_alert_query(event_severity_filter, start_time):
+    """Creates an alert query for the given severity (or severities) and start time."""
+    alert_filters = AlertSearchFilters()
+    severity = event_severity_filter
+    alert_filters.append_result(_get_severity_filter_value(severity), Severity.is_in)
+    alert_filters.append(AlertState.eq(AlertState.OPEN))
+    alert_filters.append_result(start_time, DateObserved.on_or_after)
+    alert_query = alert_filters.to_all_query()
+    return alert_query
 
 
 class Code42Client(BaseClient):
     """
-    Client will implement the service API, should not contain Demisto logic.
+    Client will implement the service API, should not contain Cortex XSOAR logic.
     Should do requests and return data
     """
 
     def __init__(self, sdk, base_url, auth, verify=True, proxy=False):
         super().__init__(base_url, verify=verify, proxy=proxy)
-        # Create the Code42 SDK instnace
-        self._sdk = sdk.create_using_local_account(base_url, auth[0], auth[1])
-        py42.settings.set_user_agent_suffix("Demisto")
+        # Create the Code42 SDK instance
+        self._sdk = sdk or py42.sdk.from_local_account(base_url, auth[0], auth[1])
+        py42.settings.set_user_agent_suffix("Cortex XSOAR")
```

Comment:
Is it ok to be changing this?

Reply:
@timabrmsn whats your opinion on this? should we leave this as is in order to keep the tracking metrics consistent, or should we change it to reduce confusion since the product has been renamed?

Reply:
I'm ok with changing it. Right now the only metrics we're seeing are from test accounts, I don't think any actual customers are using it yet. And it's simple enough to do a query to include both names if you do want to include both sets of results.